pkg/scaffold/api.go

```diff
@@ -177,6 +181,15 @@ func (api *API) scaffoldV2() error {
 		if err != nil {
 			return fmt.Errorf("error updating kustomization.yaml: %v", err)
 		}
+
+		// update scaffolded resource in project file
+		api.project.Resources = append(api.project.Resources,
+			input.Resource{Group: r.Group, Version: r.Version, Kind: r.Kind})
+		err = saveProjectFile("PROJECT", api.project)
+		if err != nil {
+			fmt.Printf("error updating project file with resource information : %v \n", err)
```

Comment:
Should it give up in this case?

Reply:
No, because this isn't *really* critical at the moment. This info just helps us to check multi-group check next time user create the resource. If this changes in future, we can make it fatal.